client/my-sites/importer/importer-medium.jsx

```diff
@@ -29,12 +30,25 @@ class ImporterMedium extends React.PureComponent {
 		);
 
 		importerData.uploadDescription = this.props.translate(
-			'Upload a {{b}}Medium export file{{/b}} to start ' + 'importing into {{b2}}%(title)s{{/b2}}.',
+			'Upload your %(importerName)s export file to start importing into ' +
```

Comment:
Any specific reason you dropped the bold text? I kind of like it as a way of calling out exactly what I need to provide to do the import.